pkg/log/logshipper/logger.go

```diff
@@ -0,0 +1,43 @@
+package logshipper
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/go-kit/kit/log"
+)
+
+const (
+	truncatedFormatString = "%s[TRUNCATED]"
+)
+
+type logger struct {
```

Comment:
I'm not sure this struct adds much. Consider integrating with logshipper

Reply:
done